MantidQt/MantidWidgets/src/InstrumentView/ObjCompAssemblyActor.cpp

```diff
@@ -28,7 +28,7 @@ namespace MantidQt
 
           ObjCompAssembly_const_sptr objAss = getObjCompAssembly();
           mNumberOfDetectors = objAss->nelements();
```

Comment:
Could you move the rhs into the assert below? The variable `mNumberOfDetectors ` isn't otherwise used, so in release mode there's no reason to call `objAss->nelements();`.